OpenRobertaRobot/src/main/java/de/fhg/iais/roberta/syntax/lang/expr/MathConst.java

```diff
@@ -42,45 +40,25 @@ private MathConst(Const mathConst, BlocklyBlockProperties properties, BlocklyCom
      * @return read only object of class {@link MathConst}
      */
     public static <V> MathConst<V> make(Const mathConst, BlocklyBlockProperties properties, BlocklyComment comment) {
-        return new MathConst<V>(mathConst, properties, comment);
+        return new MathConst<V>(BlockTypeContainer.getByName("MATH_CONST"), properties, comment, String.valueOf(mathConst));
     }
 
     /**
      * @return math constant.
      */
     public Const getMathConst() {
-        return this.mathConst;
-    }
-
-    @Override
-    public int getPrecedence() {
-        return 999;
-    }
-
-    @Override
-    public Assoc getAssoc() {
-        return Assoc.NONE;
-    }
-
-    @Override
-    public BlocklyType getVarType() {
-        return BlocklyType.NUMBER;
-    }
-
-    @Override
-    public String toString() {
-        return "MathConst [" + this.mathConst + "]";
+        return Const.get(this.mathConst);
     }
 
     /**
      * This enum defines all possible math constant.
      */
-    public static enum Const {
+    public enum Const {
```

Comment:
why is the enum not static? Is the surrounding 'this' needed?

Reply:
IntelliJ says modifier 'static' is redundant for inner enums